deeplearning4j/deeplearning4j-ui-parent/deeplearning4j-play/src/main/java/org/deeplearning4j/ui/module/defaultModule/DefaultModule.java

```diff
@@ -23,19 +23,38 @@
 import org.deeplearning4j.ui.api.Route;
 import org.deeplearning4j.ui.api.UIModule;
 import org.deeplearning4j.ui.i18n.I18NResource;
+import org.nd4j.linalg.function.Supplier;
 
 import java.io.File;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
 
+import static play.mvc.Results.ok;
 import static play.mvc.Results.redirect;
 
 /**
  * Landing page - i.e., "/" route
  * @author Alex Black
  */
 public class DefaultModule implements UIModule {
+    private boolean multiSession;
```

Comment:
Minor point: we can probably make this final...
```private final boolean multiSession```

Reply:
OK, made `DefaultModule.multiSession` final.